api/src/certification/results/domain/models/v3/Certificate.js

```diff
@@ -39,11 +40,16 @@ export class V3Certificate {
     this.deliveredAt = deliveredAt;
     this.certificationCenter = certificationCenter;
     this.pixScore = pixScore;
-    this.globalLevel = new GlobalCertificationLevel({ score: pixScore });
+    this.globalLevel = this.findLevel;
     this.verificationCode = verificationCode;
     this.maxReachableScore = MAX_REACHABLE_SCORE;
-    this.resultCompetenceTree = resultCompetenceTree;
+    this.resultCompetenceTree = this.globalLevel ? resultCompetenceTree : null;
     this.algorithmEngineVersion = algorithmEngineVersion;
     this.certificationDate = certificationDate;
   }
+
+  get findLevel() {
```

Comment:
Top, une derniere question, pourquoi passer par un getter et pas une fonction interne ?
J'ai l'impression que c'est + un nom de fonction et un usage prevu interne (les consommateurs externes utilisant globalLevel).

Par ailleurs au format getter, tu vas te retrouver avec une propriete 'findLevel' sur certaines iterations sur le parcours des attributs de la classe.

```suggestion
  #findLevel() {
```

Reply:
Effectivement, aucune raison à ce qu'il s'agisse d'un getter dans ce cas. Je change.